addons/crm/models/crm_lead.py

```diff
@@ -465,7 +465,7 @@ def write(self, vals):
         if 'stage_id' in vals:
             stage_id = self.env['crm.stage'].browse(vals['stage_id'])
             if stage_id.is_won:
-                vals.update({'probability': 100})
+                vals.update({'probability': 100, 'date_deadline': fields.Datetime.now()})
```

Comment:
Overlap with date_closed -> makes no sense to set a deadline automatically

Reply:
To drop